src/lib/discord/discordWebhookWorker.js

```diff
@@ -63,7 +63,7 @@ class DiscordWebhookWorker {
 					if (!res.headers.via) {
 						this.logs.discord.error(`${senderId} WEBHOOK 429 Rate limit [Discord Webhook] TELL @JABES ON DISCORD THIS COULD BE FROM CLOUDFLARE: ${retryAfterMs}`)
 					}
-					await this.sleep(retryAfterMs + Math.random() * 5000)
+					await this.sleep(+retryAfterMs + Math.random() * 5000)
```

Comment:
won't fix the problem, you're expecting parseInt(undefined) * 5000 is better than undefined * 5000
both cases parse to NaN

should be something like `await this.sleep(retryAfterMs? retryAfterMs + Math.random() * 5000) : 5000`

Reply:
That would be safer, but retryAfterMs is present and is a string in the case was solving for

Reply:
weird, res.headers['retry-after'] was not undefined but some string, what string?

Reply:
1059 or some such... as expected. So actually ended up waiting 10593000 ms or something like that

Reply:
what I mean -> 
![image](https://user-images.githubusercontent.com/23346150/119730141-0db42a80-be7e-11eb-9cfc-818880d95b70.png)

Reply:
and the other warning there is because I read that if cloudflare sends the 429 not discord it is in seconds not ms (spec says seconds but discord have it wrong).  But I have never seen one ;-)

Reply:
+String + otherMath would only work if it's an _empty_ string

+var is short for parseInt(var)